.github/workflows/forceMergePRBypassAudit.yml

```diff
@@ -0,0 +1,82 @@
+name: Force-Merge PR (Bypass Audit Requirement)
+# - This git action may only be used in exceptional cases
+# - it can only be executed by the CTO or the Information Security Manager/Architect
+# - a valid reason must be provided in order to force-merge a given PR
```

Comment:
Adding some examples would help to understand in which cases we plan to use that. 
- mainly automated actions expect and audit/additional review but the changes are only in comments or other not security relevant code

Reply:
added